components/HamburgerMenu.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+    <div class="cursor-pointer w-full flex" v-close-on-outside-click>
+        <SVGHamburgerMenuIcon id="hamburger-menu-icon" @click="toggleMenu()" alt="hamburger menu" class="h-8 w-8" />
+        <div id="hamburger-menu" v-show="isMenuOpen"
+            class="z-20 absolute top-0 left-0 h-full w-2/3 pt-6 pb-8 rounded bg-primary-bg border-2 flex flex-col justify-between">
+            <div id="upper-section">
+                <div id="header" class="flex justify-between px-5">
+                    <div class="flex">
+                        <div class="title-text flex flex-col flex-shrink-0" data-testid="logo">
+                            <div class="text-lg text-primary-text group-hover:text-primary-hover font-semibold">
+                                Find a Doc, Japan
+                            </div>
+                        </div>
+                    </div>
+                    <button id="close-button" @click="closeMenu()" class="px-2 py-.5 mr-2">
+                        <svg class="stroke-primary" width="20" heigh="20" viewBox="4 0 15 25">
+                            <path stroke-width="3" fill="none" d="M6.25,6.25,17.75,17.75" />
+                            <path stroke-width="3" fill="none" d="M6.25,17.75,17.75,6.25" />
+                        </svg>
+                    </button>
+                </div>
+                <div id="language-section" class="flex mt-2 px-5">
+                    <span class="text-primary-text self-center mr-2">{{ $t('hamburgerMenu.languageDropdownTitle') }}</span>
+                    <LocaleSelector class="py-1.5 text-primary-text bg-primary-bg" />
+                </div>
+                <div id="menu-items" class="mt-10 px-5 flex flex-col gap-6">
+                    <NuxtLink :to="'/'">
+                        <div @click="toggleMenu()">
+                            {{ $t('hamburgerMenu.home') }}
+                        </div>
+                    </NuxtLink>
+                    <NuxtLink :to="'/about'">
+                        <div @click="toggleMenu()">
+                            {{ $t('hamburgerMenu.about') }}
+                        </div>
+                    </NuxtLink>
+                    <NuxtLink :to="'https://forms.gle/4E763qfaq46kEsn99'" target="_blank">
+                        <div @click="toggleMenu()">
+                            {{ $t('hamburgerMenu.contact') }}
```

Comment:
This string needs to be added to `ja.json`. 
<img width="223" alt="Screenshot 2024-06-03 at 8 10 44 AM" src="https://github.com/ourjapanlife/findadoc-web/assets/31802656/3032dc7b-e181-44c0-84e8-5f146203620a">

Reply:
done ✅